### DISCUSSION TITLE
Entity not showing up after installation

### DISCUSSION THREAD

**Discussion by ernolddd** [hultenvp/solis-sensor#154] [Q&A]
Hello!

I have a Sofar Solar inverter that uses the  Solarman platform for web-interface. Also, the same data is available with the same credentials at m.ginlong.com

I have installed the solis-sensor with HACS, rebooted HASS, added the required lines to configuration.yaml, rebooted again but I cant see the sensor available for Energy dashboard or to Add as a Card. 

I have played around with the conf.yaml and tried to switch from home.solarm.cn to Ginlong etc, but no change.

How have you managed to install the sensor? What might I be doing wrong?

 I would think I get the sensor showing, even with no information, but nothing is showing up.

**Comment by ernolddd** [comment]
Nevermind.

I had already a sensor: line in my configfile, and you cant´t have multiple sensors like that in YAML.

I fixed this having one sensor with multiple platforms. everything is ok now.

**Reply by YngvarT** [reply]
Hi. I am struggling with the same. 
What lines do you have in your config.yaml file 

Am I missing something? 

  - platform: solis
    name: My Solis Inverter
    portal_domain: www.soliscloud.com:13333
    portal_username: mail@mail.com
    portal_key_id: 1___________431
    portal_secret: b94c0____________3a9
    portal_plant_id: 46____1
